config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+common:
+  ssh_key_name: vaijab
+  coreos_ami_name: CoreOS-beta-695.0.0-hvm
+  coreos_min_instances: 3
+
+prod:
+  vpc_name: prod-vpc
+  infra_stack_name: prod-infra
+dev:
+  vpc_name: dev-vpc
+  infra_stack_name: dev-infra
+  etcd_discovery_url: https://discovery.etcd.io/2408dbac5b15811a8585d57931d9f001
```

Comment:
Probably here by mistake? we should add a `.gitignore`

Might be good to have a template too as an example.